### ISSUE TITLE
DBeaver 3.8.4 crashes when selecting a schema

### ISSUE THREAD

**Issue by ced75** [serge-rider/dbeaver#1173]
Hi,

Linux Mint 17.1
DBeaver 3.8.4
PostgreSQL 9.6.1

When I start DBeaver, I am automatically connecting to my last PostgreSQL connection.
In the menu, I see the connection selection shortcut and the schema selection shortcut as this:
![image](https://cloud.githubusercontent.com/assets/12892581/22197034/75ce76e4-e150-11e6-9008-142c5ea929d3.png)
If I clic twice on the arrow on the schema selection shortcut, DBeaver crashes without any error message.
I had a look in the error log, but nothing is written.

I hope you can reproduce it.
Regards,

Cedric

**Comment by serge-rider** [comment]
Can't reproduce that on Ubuntu. Please check for any `hs_err_pid*.log` files in DBeaver install folder or in current directory. They may contain crash report.

What happens if you click once? Does schema combo expand?

**Comment by ced75** [comment]
OK, this is effectively hard to reproduce, but I still can do it.
If I clic on the arrow, the schema combo expands correctly. If then I clic on a schema, it works fine.

The problem seems to occure after a first start of DBeaver, if I clic on the combo arrow twice.

I have found several hs.err.pid*.log files. 
I attach several of them
[hs_err_pid_DBeaver.zip](https://github.com/serge-rider/dbeaver/files/723295/hs_err_pid_DBeaver.zip)

Hope this will help... Anyway, this is a minor bug that I found "by accident".
